### ISSUE TITLE
Meta content with URL

### ISSUE THREAD

**Issue by brunowego** [hurrymaplelad/grunt-relative-root#11]
Think this case below, have a solution for this? Thanks.

```html
<!--[if lte IE 9]>
  <meta http-equiv="refresh" content="0;url=/not-supported.html">
<![endif]-->
```

**Comment by hurrymaplelad** [comment]
@brunowego welome!  This plugin is just a [collection of regular expression](https://github.com/hurrymaplelad/grunt-relative-root/blob/master/tasks/relative_root.js#L30).  Feel free to add one that covers this case, along with a [test case](https://github.com/hurrymaplelad/grunt-relative-root/blob/master/test/fixtures/prudent.html) in a PR.

**Comment by hurrymaplelad** [comment]
Fixed in #12
